service/frontend/accessControlledWorkflowHandler.go

```diff
@@ -0,0 +1,725 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
[not a blocker ] filename seems rather long - I don't have a suggestion but if you think of a way to shorten it without compromising clarity, go for it.